<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="orbitallabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Orbital Calculations</string>
    </property>
   </widget>
   <widget class="QLabel" name="basicdeltavlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Basic Delta-v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="basicdvclearbutton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="twrlabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TWR Calculations</string>
    </property>
   </widget>
   <widget class="QLabel" name="orbitalvelocitylabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Orbital Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="inclichangelabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inclination Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twrclearbutton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="orbvelclearbutton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incchangeclearbutton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>284</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="orbitalvelocitycombo">
       <item>
        <property name="text">
         <string>Choose the Orbital Body</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moho</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gilly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minmus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Duna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ike</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dres</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Laythe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tylo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eeloo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="orbitalvelocityheight">
       <property name="placeholderText">
        <string>Height of your orbit in km</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="orbitalvelocitybutton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="orbitalvelocityanswer"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>83</y>
      <width>248</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mstartlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mass (start)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="basicdvresultlineedit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mendlineedit">
       <property name="placeholderText">
        <string>in tonnes</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="basicdeltavtotalbutton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add to Total:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mendlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mass (end)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="basicdvbutton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="basicdeltavtotalresult"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="basicdeltavcombo">
       <item>
        <property name="text">
         <string>Choose the engine in use</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-1R</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24-77</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mk-55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48-7S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-909</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-T30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-T45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CR-7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poodle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skipper</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mainsail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LFB KR-1x2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KR2L+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S3 KS-25x4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RT-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RT-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BACC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S1 SRB-KD25k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sepratron I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PB-ION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RV-105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Place-Anywhere 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vernor Engine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J-33 Basic Jet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JX-4 Turbo Ramjet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CR-7 Jet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mstartlineedit">
       <property name="placeholderText">
        <string>in tonnes</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_atm_or_vac">
       <property name="text">
        <string>Use Athmospheric properties</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_atm_or_vac">
       <property name="text">
        <string>Athmospheric</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>61</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="twrmasslineedit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="twrengineamount">
       <property name="placeholderText">
        <string>Amount of engines</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="twrmasslabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mass of Rocket</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="twrtonlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="twrcalculatebutton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="twrcombo">
       <item>
        <property name="text">
         <string>Choose the engine in use</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-1R</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24-77</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mk-55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>48-7S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-909</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-T30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-T45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CR-7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LV-N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poodle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skipper</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mainsail</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LFB KR-1x2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KR2L+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S3 KS-25x4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RT-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RT-10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BACC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S1 SRB-KD25k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sepratron I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PB-ION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RV-105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Place-Anywhere 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vernor Engine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J-33 Basic Jet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JX-4 Turbo Ramjet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CR-7 Jet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QComboBox" name="twrbodycombo">
       <item>
        <property name="text">
         <string>Choose the Orbital Body</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moho</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gilly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minmus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Duna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ike</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dres</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Laythe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tylo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bop</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eeloo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="twrresultlineedit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_twr_atm_or_vac">
       <property name="text">
        <string>Use Athmospheric properties</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkBox_twr_atm_or_vac">
       <property name="text">
        <string>Athmospheric</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>333</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="incstartlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Inc (start)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="incstartlineedit">
       <property name="placeholderText">
        <string>in degrees</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="incchangedeglabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DEG</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="incendlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Inc (end)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="incendlineedit">
       <property name="placeholderText">
        <string>in degrees</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="incchangedeglabel_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DEG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="incchangevelocitylabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Orbital velocity</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="incchangevelocitylineedit">
       <property name="placeholderText">
        <string>in km/s</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="incchangekmslabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Km/s</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="incchangeresultbutton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="incchangeresultlineedit">
       <property name="placeholderText">
        <string>in km/s</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLabel" name="incchangekmslabel_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Km/s</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>191</y>
      <width>319</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Geostationary orbit altitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_geostat_orbit">
       <item>
        <property name="text">
         <string>Choose the Orbital Body</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moho</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kerbin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Duna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dres</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eeloo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="geostatlineedit">
       <property name="placeholderText">
        <string>Height of the orbit in km</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="geostatkmlabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Km</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
